src/odemis/driver/pigcs.py

```diff
@@ -2037,8 +2052,6 @@ def _startServo(self, axis):
                 self.SetParameter(axis, 0x56, 1, check=False)  # 1 = on
                 time.sleep(2)  # 2 s seems long enough for the encoder to initialise
             self.SetServo(axis, True)
-            # To allow (relative) moves, even if it's not actually referenced
-            self.SetReferenceMode(axis, False)
```

Comment:
Without this, it'll be impossible to move again the hardware. So it should stay as-is. In your case, is this code actually called (ie, is `self._servo_suspend` True or False?). If so, we probably want to adjust the computation of `_servo_suspend` to be always False if the axis is referenceable. 
Note that this code might still be called in case some error happened during a move (eg, reached a limit, or it failed to reach the target position). I'd suggest that for now we reset the referencing state to False, for safety. If it turns out some some error can be handled without resetting the referencing, we could handle them explicitly, when we face them.

Reply:
For now the `_servo_suspend` is always False if we are in "read" mode and True if in "full" mode. So would you suggest to have a differentiation in "full" mode between referenceable and not referenceable?

Reply:
I think we should just always call `self.SetReferenceMode(axis, False)` here. If we just started the servo, either it's because we just initialised, so the axis is not referenced yet, or there was a stop servo, so it's not referenced. The `startReference()` function will switch the mode when needed.